feat: add pull request template guide (#13)
SRGSSR/guilde-plateformes-propres: public/guides/PULL_REQUEST_GUIDE.md

  approaches, coding styles, promoting a culture of continuous learning within the team.
- **Bug Prevention:** A fresh set of eyes often catches issues that the original author might have
  overlooked.

## The Pull Request Template

The pull request template is a valuable tool for creating a successful pull request. It is a guide
to help contributors understand what is expected of them.

Here are some reasons why adding a pull request template to your repository is important:

- **Clarity and Consistency**: A pull request template ensures that every contributor provides the
  necessary information when proposing changes.
- **Efficiency**: With a template, reviewers don't have to ask basic questions about the pull
  request, as most of the essential details should be included in the template.
- **Documentation**: Detailed pull requests serve as a form of documentation, providing insight into
  the project's history and the rationale behind certain decisions.
- **Quality Control**: A checklist in the template can serve as a reminder for contributors to
  adhere to the project's standards and guidelines, leading to higher quality submissions.

### Adding a Pull Request Template

> You can store your pull request template here: `.github/pull_request_template.md`. You can review
> other options on the following GitHub documentation:
> 
> [Creating a pull request template for your repository][github-creating-pr-template].

Reviewer: ```suggestion
You can store your pull request template in `.github/pull_request_template.md`.
You can review other options on the [GitHub documentation][github-creating-pr-template].
```
